routemaster/controllers/subscriber.rb

```diff
@@ -23,8 +23,7 @@ class Subscriber < Sinatra::Base
         end
 
         topics = data['topics'].map do |name|
-          Models::Topic.find(name) ||
-          Models::Topic.new(name: name, publisher: nil)
+          Models::Topic.find_or_create!(name: name)
```

Comment:
My apologies. I was thinking of the `topic_controller`, and created a lot of confusion.
Yeah, in this case `Topic.find_or_create!` cannot raise any (of our) exeptions; so any exception raised (e.g. Redis connection-related) would bubble up and validly (I think) result in a 500.